src/gsy_e/models/strategy/heat_pump.py

```diff
@@ -84,6 +84,19 @@ def __init__(self,
 
         self.preferred_buying_rate = preferred_buying_rate
 
+    @staticmethod
+    def deserialize_args(constructor_args: Dict) -> Dict:
+        """Deserialize the constructor arguments for the HeatPump strategy."""
```

Comment:
I assume that the other arguments do not need to be deserialized because they either do not need to modify their type during deserialization, or they have exactly the same name in gsy-e and gsy-web right? In this case looks good to me

Reply:
Yes. Everything else is in good shape.